HintList.py

```diff
@@ -168,6 +181,19 @@ def tokens_required_by_settings(world):
     'OGC Great Fairy Reward':       lambda world: world.settings.shuffle_interior_entrances == 'off',
 }
 
+# Some dual hints should only be enabled under certain settings
+conditional_dual = {
+    'LW Deku Theater Rewards':                                  lambda world: world.settings.complete_mask_quest,
+    'GV Pieces of Heart Ledges':                                lambda world: not world.settings.shuffle_cows and not world.settings.tokensanity in ['overworld', 'all'],
```

Comment:
```suggestion
    'GV Pieces of Heart Ledges':                                lambda world: not world.settings.shuffle_cows and world.settings.tokensanity not in ['overworld', 'all'],
```

Same for the others.